packages/@aws-cdk/aws-ivs/lib/channel.ts

```diff
@@ -0,0 +1,172 @@
+  /**
+   * The channel ARN. For example: arn:aws:ivs:us-west-2:123456789012:channel/abcdABCDefgh
+   *
+   * @attribute
+   */
+  readonly channelArn: string;
+
+  /**
+   * Adds a stream key for this IVS Channel.
+   * @param id construct ID
+   */
+  addStreamKey(id: string): StreamKey;
+}
+
+/**
+ * Reference to a new or existing IVS Channel
+ */
+abstract class ChannelBase extends core.Resource implements IChannel {
+  public abstract readonly channelArn: string;
+
+  public addStreamKey(id: string): StreamKey {
+    return new StreamKey(this, id, {
+      channel: this,
+    });
+  }
+}
+
+/**
+  Channel latency mode
+*/
+export enum LatencyMode {
+  /**
+   * Use LOW to minimize broadcaster-to-viewer latency for interactive broadcasts.
+   */
+  LOW = 'LOW',
+
+  /**
+   * Use NORMAL for broadcasts that do not require viewer interaction.
+   */
+  NORMAL = 'NORMAL',
+}
+
+/**
+  * The channel type, which determines the allowable resolution and bitrate.
+  * If you exceed the allowable resolution or bitrate, the stream probably will disconnect immediately.
+*/
+export enum ChannelType {
+  /**
+   * Multiple qualities are generated from the original input, to automatically give viewers the best experience for
+   * their devices and network conditions.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ivs-channel.html
+   */
+  STANDARD = 'STANDARD',
+
+  /**
+   * delivers the original input to viewers. The viewer’s video-quality choice is limited to the original input.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ivs-channel.html
+   */
+  BASIC = 'BASIC',
+}
+
+/**
+ * Properties for creating a new Channel
+ */
+export interface ChannelProps {
+  /**
+   * Whether the channel is authorized.
+   *
+   * If you wish to make an authorized channel, you will need to ensure that
+   * a PlaybackKeyPair has been uploaded to your account as this is used to
+   * validate the signed JWT that is required for authorization
+   *
+   * @default false
+   */
+  readonly authorized?: boolean;
+
+  /**
+   * Channel latency mode.
+   *
+   * @default LatencyMode.LOW
+   */
+  readonly latencyMode?: LatencyMode;
+
+  /**
+   * Channel name.
+   *
+   * @default - None
+   */
+  readonly name?: string;
+
+  /**
+   * The channel type, which determines the allowable resolution and bitrate.
+   * If you exceed the allowable resolution or bitrate, the stream will disconnect immediately
+   *
+   * @default ChannelType.STANDARD
+   */
+  readonly type?: ChannelType;
+}
+
+/**
+  A new IVS channel
+*/
+export class Channel extends ChannelBase {
+  /**
+   * Import an existing channel
+   */
+  public static fromChannelArn(scope: Construct, id: string, channelArn: string): IChannel {
+    // The astute among you may notice that this regex differs from the documentation
+    // This is because the documentation is wrong :)
```

Comment:
how was this verified?
perhaps we can correct the documentation - this comment is bound to go stale

Reply:
I work on the IVS team so, I guess I just know. Docs are still wrong but will follow up again.